experimental/templates/multiplatform-template/README.md

```diff
@@ -1,8 +1,58 @@
-Compose Multiplatform Application
+# Compose Multiplatform Application
 
-**Desktop**
+Сan run on desktop, Android and iOS.
+Note that iOS target is experimental so far.
+
+*Prerequisites*: to run on iOS, you should have "Kotlin Multiplatform Mobile" plugin installed 
+                 in Android Studio with [installed CocoaPods](https://kotlinlang.org/docs/native-cocoapods.html).
+
+## How to run
+
+Choose a run configuration for an appropriate target in Android Studio and run it.
+
+## Configuring iOS application and running it on a real iOS device
+
+*Before trying iOS target we strongly recommend
+to create a sample iOS project in XCode and
+manage to run it on a real iOS device.*
+
+To deal with iOS target, this project contains `iosApp` folder 
+with XCode project (`iosApp/iosApp.xcodeproj`) and entry point 
+Swift source files (`iosApp/iosApp`). 
+However, to not interact with XCode directly, settings that you need 
+to configure are extracted to `iosApp/Configuration/Config.xcconfig`, namely:
+
+- `APP_NAME` - target executable and application bundle name
+- `BUNDLE_ID` - [bundle identifier](https://developer.apple.com/documentation/appstoreconnectapi/bundle_ids)
+- `TEAM_ID` - [Team ID](https://developer.apple.com/help/account/manage-your-team/locate-your-team-id/#:~:text=A%20Team%20ID%20is%20a,developer%20in%20App%20Store%20Connect.)
+
+Note that you need to configure `TEAM_ID` at least to be able 
+to run on a real iOS device.
+
+**Hint:** If you managed to run a sample iOS project in XCode on a real iOS device as we suggested above, 
```

Comment:
>Does this hint mean that we need to open .xcodeproj of the created standard Xcode sample? 

yes

>What if people don't have team id registered? 

They will not be able to run on a real iOS device

>Isn't https://developer.apple.com/help/account/manage-your-team/locate-your-team-id/ enough to find it?

It was not enough for me.  The link showed me JetBrains Team ID that I could not use to register a new bundle identifier (only admins of a company account can create new bundle id). XCode allowed me to sign a sample app only with my Personal Team ID that I was not able to find anywhere (as XCode does not reveal it explicitly). 
I even contacted Apple support with the subject with no good explanation. However if you are able to sign an app with personal Team ID (my case) you can easily find its value within xcodeproj -- maybe it can be useful to somewhere else.